docs/docsite/rst/user_guide/playbooks_async.rst

```diff
@@ -9,6 +9,16 @@ be running operations that take longer than the SSH timeout.
 
 To avoid blocking or timeout issues, you can use asynchronous mode to run all of your tasks at once and then poll until they are done.
 
+The behaviour of asynchronous mode depends on the value of `poll`.
+
+
+Avoid connection timeouts: poll > 0
+-----------------------------------
+
+When `poll` is a positive value, the playbook will *still* block on the task until it either completes, fails or times out.
+
+In this case, however, `async` explicitly sets the timeout you wish to apply to this task rather than being limited by the connection method timeout.
+
 To launch a task asynchronously, specify its maximum runtime
 and how frequently you would like to poll for status.  The default
 poll value is 10 seconds if you do not specify a value for `poll`::
```

Comment:
15? https://docs.ansible.com/ansible/latest/reference_appendices/config.html#default-poll-interval

Reply:
`DEFAULT_POLL_INTERVAL` ony affects 'adhoc'i.e `ansible -a 'find /' --async`
while 10s is default for `async` in plays'

Reply:
I've updated the sentence to say 15.